src/components/multilineBox/examples/MultilineBoxExamples.tsx

```diff
@@ -23,6 +24,43 @@ const MultilineBoxWithContainer = _.compose(
     }),
 )(MultilineBox);
 
+const MultilineBoxWithRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+)(MultilineBox);
+
+const MultilineBoxWithContainerAndRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+    multilineBoxContainer({
+        container: {
+            tag: 'div',
+            props: {
+                className: 'p1',
+            },
+        },
+    }),
+)(MultilineBox);
+
+const MultilineBoxWithContainerAndTwoRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+    multilineBoxContainer({
+        container: {
+            tag: 'div',
+            props: {
+                className: 'mod-border p1',
+            },
+        },
+    }),

```

Comment:
I'm looking at this and wouldn't it be easier to directly set the element like this:

```typescript
const MultilineBoxWithContainerAndTwoRemoveButton = _.compose(
    multilineBoxWithRemoveButton(),
    multilineBoxContainer((props: theCorrectType) => (<div className='mod-border p1'>{props.children}</div>),
...
```

Reply:
Updated with a global variable for each example